cypress/integration/menu/position.feature

```diff
@@ -1,42 +1,46 @@
+        When the menu is opened
+        Then the menu is below the anchor
         And the menu is left aligned with the anchor
 
     Scenario: Flipped vertically
-        Given the anchor has less than 368px space below, before the end of the <body> element
-        When the menu is rendered
-        Then the menu is rendered above the anchor
+        Given there is not enough space below the anchor to fit the default maximum
+        When the menu is opened
+        Then the menu is above the anchor
         And the menu is left aligned with the anchor
 
-    Scenario: Less than 368px available space below and above anchor
-        Given the anchor has less than 368px space above and below before the limits of the <body> element
-        When the menu dropdown is rendered
-        Then it is rendered below the anchor
-        And the height of the menu is reduced to fit within the <body> element
-        But the heght of the menu is not reduced below 50px (and what if it is)?****
+    Scenario: Less than 368px and more than 50px available space below and above anchor
+        Given there is not enough space above and below the anchor to fit the default maximum
+        But there is more than 50px of available space
+        When the menu is opened
+        Then the menu is below the anchor
+        And the height of the menu is reduced to fit
+
+    # ¯\_(ツ)_/¯
+    # This will cause the menu always to be off screen, but that's the apps fault
+    Scenario: Less than 50px available space below and above anchor
+        Given there is not enough space above and below the anchor to fit the default maximum
+        When the menu is opened
+        Then the menu is below the anchor
+        And the heght of the menu is not reduced below 50px
 
     Scenario: Flipped horizontally
-        Given the anchor has less than 128px space to the right, before the limit of the <body> element
-        When the menu is rendered
-        Then the menu is rendered below the anchor
-        And the menu is right aligned with the anchor
+        Given the anchor has less than 128px space to the left
+        When the menu is opened
+        Then the menu is right of the anchor
+        And the menu is below the anchor
 
+    # ¯\_(ツ)_/¯
+    # This will cause the menu always to be off screen, but that's the apps fault
     Scenario: Forced body overflow
-        Given the anchor has less than 128px space to the left and the right, before the limit of the <body> element
-        When the menu is rendered
-        Then the menu is rendered below the anchor
-        And the menu is left aligned with the anchor
-        And the menu overflows the body element
-        # ¯\_(ツ)_/¯
-        # The app would need to handle this, this is beyond the scope of the positioning utility
+        Given the anchor has less than 128px space to the left and the right
+        When the menu is opened
+        Then the menu is left of the anchor
```

Comment:
'the menu is left aligned...' is the phrasing used in previous scenario, we should be consistent here.